### PR TITLE
Revised GridSets & docs

### PR DESCRIPTION
**Pull request by quaxalber**
Adapted UK and US to the new layout. If you can please test and see if you spot any issues.

### CODE REVIEW

File: docs/en/README.md
Comment (quaxalber) [inline docs/en/README.md:1]: Added footnotes and made some changes for better clarity and accuracy. Please let me know what you think and I will adapt for EN and DE

File: deamon - debug.bat
Comment (quaxalber) [inline deamon - debug.bat:1]: This allows to easily run a daemon on Windows without RelayKeys hardware

Comment (willwade) [reply deamon - debug.bat:1]: Great. I did try and tidy up all this non-essential testing code once to tidy up the repo but never managed it. I wonder about some kind of multi-batch script you call with flags. I don’t know. Adding more work than it needs maybe. 

Makes sense the Xfrozen_modules=off thing

**Comment by quaxalber** [comment]
Green lights for a merge from my side

File: deamon - debug.bat
Comment (quaxalber) [reply deamon - debug.bat:1]: I hear you. I reckon another tidying/refactoring pass would be time well spent since (inevitably) quite some technical debt has accumulated over time (long functions, repetitive code, deep nesting, ...) which makes it hard to follow along and make changes for someone new to the project. @willwade
